packages/react-components/react-portal/src/components/Portal/usePortalMountNode.ts

```diff
@@ -56,6 +56,8 @@ export const usePortalMountNode = (options: UsePortalMountNodeOptions): HTMLElem
 
       element.classList.add(...classesToApply);
       element.setAttribute('dir', dir);
+      element.setAttribute('data-portal-node', 'true');
```

Comment:
Let's document this here: https://react.fluentui.dev/?path=/docs/components-portal-portal--default

Reply:
<img width="1017" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/b969883b-d86a-46dc-bc75-2552b0010ec3">

@ling1726 updated docs.